migrations.md

```diff
@@ -3,24 +3,30 @@
 ### Add database
 
 in ./alembic/env.py
-'''
-DATABASE_DIR = <location of database.db>
-config.set_main_option('sqlalchemy.url', 'sqlite:///{}/database.db'.format(DATABASE_DIR))
-'''
+
+    DATABASE_DIR = <location of database.db>
+    config.set_main_option('sqlalchemy.url','sqlite:///{}/database.db'.format(DATABASE_DIR))
+
+
+
 
 Now you can make changes in db_model.
```

Comment:
Currently hydrus creates db_model with the assistance of doc_parse based on the Hydra API provided by the user.  That said, I don't think that's quite the use case we would have in the Hydra Ecosystem. I'll comment below how I personally think that could be done.